src/components/StaticInfo.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import { StyleSheet } from 'react-native';
+
+import FullWidthContainer from './FullWidthContainer';
+import LargeText from './LargeText';
+
+import recycleFacts from '../data/recycling-facts.json';
+
+class StaticInfo extends Component {
+  insertRandomFact() {
```

Comment:
nit: it's not "insert"ing anything, it's just getting one

Reply:
I can update this tonight.